src/NServiceBus.Core/Transports/Msmq/MsmqTransportInfrastructure.cs

```diff
@@ -81,20 +81,18 @@ public override TransportReceiveInfrastructure ConfigureReceiveInfrastructure()
         {
             new CheckMachineNameForComplianceWithDtcLimitation().Check();
 
-            var builder = connectionString != null
-                ? new MsmqConnectionStringBuilder(connectionString).RetrieveSettings()
```

Comment:
Good catch @mikeminutillo , fixed in https://github.com/Particular/NServiceBus/pull/4343/commits/3808e042b2916a6a2a5eb3d3b3fd384eca0b3cda